### PR TITLE
Add the navbar

### PR DESCRIPTION
**Pull request by martinremis**
## Summary
Add a Navbar.

## Screenshot
navbar1

## Trello Card
https://trello.com/c/qSJPvx8d/13-navbar

### CODE REVIEW

File: scss/application.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
+$big-text: 24px;
```
Comment (FrankIglesias) [inline scss/application.scss]: you can define this variables inside a `text.scss` file

File: scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.navbar {
+  background-color: white;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: use a variable

File: scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	left: 0;
+	position: fixed;
+	top: 0;
+	width: 100%;
+}
+
+.nav-brand {
+  align-self: center;
+  height: 45px;
+  margin: 2em;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: use px

File: scss/styles.scss [hunk 4]
```diff
@@ -2,6 +2,10 @@
   font-family: 'Roboto', sans-serif;
 }
 
+body {
```
Comment (FrankIglesias) [inline scss/styles.scss]: The body should not have any style applied, define a container at least for the layout. maybe you can change this for a margin top to another component

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
Great job definin variables for sizes, just a few changes!

File: scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+.nav-brand {
+  align-self: center;
+  height: 45px;
+  margin: 32px;
+}
+
+.nav-title {
+  align-self: center;
+  font-size: $big-text;
+  margin: 1em;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: use px instead

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
Check indentation

File: scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
+.navbar {
+  background-color: $white;
```
Comment (FrankIglesias) [inline scss/navbar.scss:2]: check indentation

**Comment by martinremis** [comment]
Navbar new capture
navbar3

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**